packages/airnode-node/src/handlers/call-api.ts

```diff
@@ -103,7 +135,7 @@ function verifySponsorWallet(payload: CallApiPayload): LogsData<ApiCallErrorResp
 
 function verifyRequestId(payload: CallApiPayload): LogsData<ApiCallErrorResponse> | null {
   const { aggregatedApiCall } = payload;
-  if (aggregatedApiCall.type === 'testing-gateway') return null;
+  if (aggregatedApiCall.type !== 'regular') return null;
```

Comment:
I am not sure how should this change in regards to the PSP subscriptionId in contracts, so I skipped it for the 2nd party endpoint (it's only performed for regular rrp requests)